<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TRL</class>
 <widget class="QWidget" name="TRL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multiline TRL Calibration</string>
  </property>
  <layout class="QVBoxLayout" name="tab_calStandards_layout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9"/>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_TriggerNew">
       <property name="text">
        <string>Trigger New</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Network Analyzer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_analyzer">
         <item>
          <property name="text">
           <string>none</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Visa String</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_visaString"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_calStandards">
       <attribute name="title">
        <string>Cal Standards</string>
       </attribute>
       <layout class="QVBoxLayout" name="tab_calStandards_layout">
        <property name="margin">
         <number>8</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Thru</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_measureThru">
            <property name="text">
             <string>Measure</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_loadThru">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Switch Terms</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_measureSwitchTerms">
            <property name="text">
             <string>Measure</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_loadSwitchTerms">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="NetworkListWidget" name="listWidget_thru"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="tab_calStandards_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Reflect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_measureReflect">
            <property name="text">
             <string>Measure</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_loadReflect">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="NetworkListWidget" name="listWidget_reflect"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Line</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_measureLine">
            <property name="text">
             <string>Measure</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_loadLine">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="NetworkListWidget" name="listWidget_line"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QPushButton" name="btn_saveCalibration">
            <property name="text">
             <string>Save Cal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_loadCalibration">
            <property name="text">
             <string>Load Cal</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_measurements">
       <attribute name="title">
        <string>Measurements</string>
       </attribute>
       <layout class="QVBoxLayout" name="tab_measurements_layout">
        <property name="margin">
         <number>8</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="btn_measureMeasurement">
            <property name="text">
             <string>Measure</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_loadMeasurement">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_calibrate">
            <property name="text">
             <string>Calibrate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="NetworkListWidget" name="listWidget_measurements"/>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Save Options</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QRadioButton" name="radio_saveRaw">
               <property name="text">
                <string>Save Raw</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radio_saveCal">
               <property name="text">
                <string>Save Cal</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radio_saveBoth">
               <property name="text">
                <string>Save Both</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="btn_saveSelectedMeasurements">
               <property name="text">
                <string>Save Selected</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_saveAllMeasurements">
               <property name="text">
                <string>Save All</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_plotArea"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NetworkListWidget</class>
   <extends>QListWidget</extends>
   <header>pyMultiCal.widgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
